### ISSUE TITLE
Feature Request.  Turn on keyboard capture by default for connection

### ISSUE THREAD

**Issue by slcpunk** [FreeRDP/Remmina#1041]
The first thing I always do after connecting is hit "right-control" to start capturing the keyboard.  Could this be a parameter of the connection/setting?  Constantly hitting Alt-Tab and having it switch back to my desktop instead of the next window in the RDP session.

**Comment by giox069** [comment]
Hi @slcpunk. In remmina 1.2 the status of the keyboard capture is saved for every connection profile.
When you connect using a profile (not quick connect), and then enable the keyboard capture, the keyboard capture status will be saved on the profile for all subsequent connection.
Are you using quick connect feature ?

**Comment by slcpunk** [comment]
No, I use saved connections.  setting doesn't seem to stick for
      me I guess?  every time I connect, the capture is off and I have
      to turn it on.  ( press right-ctrl )
    
    I am running Remmina 1.2.0-rcgit-15 (git n/a) on Ubuntu 16.04
    
    I connect to a number of servers running 2012 via RDP.
    
    Thanks for the reply.
    
    Shane
    
    
    On 11/06/2016 02:28 AM, Giovanni
      Panozzo wrote:
    
    
      Hi @slcpunk. In remmina 1.2 the status
        of the keyboard capture is saved for every connection profile.
        When you connect using a profile (not quick connect), and then
        enable the keyboard capture, the keyboard capture status will be
        saved on the profile for all subsequent connection.
        Are you using quick connect feature ?
      —
        You are receiving this because you were mentioned.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/FreeRDP/Remmina","title":"FreeRDP/Remmina","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/FreeRDP/Remmina"}},"updates":{"snippets":[{"icon":"PERSON","message":"@giox069 in #1041: Hi @slcpunk. In remmina 1.2 the status of the keyboard capture is saved for every connection profile.\r\nWhen you connect using a profile (not quick connect), and then enable the keyboard capture, the keyboard capture status will be saved on the profile for all subsequent connection.\r\nAre you using quick connect feature ?"}],"action":{"name":"View Issue","url":"https://github.com/FreeRDP/Remmina/issues/1041#issuecomment-258669392"}}}

**Comment by giox069** [comment]
@slcpunk: remmina profiles files are stored in $HOME/.local/share/remmina/ (or $HOME/.remmina for backward compatibility). They are files with the ".remmina" extension, and they should be updated when a connection is regularly closed (not when killing remmina).
Try to find the correct .remmina file that corresponds to your connection, and open it with a text editor. You should find a line called "keyboard_grab=0" or 1. Check if it changes after connecting, enabling keyboard capture, and then disconnecting or closing the connection window.

**Comment by slcpunk** [comment]
For the Server 2012 connection, there was no entry for
      "keyboard_grab" in the profile. Very strange.  ( I connected, hit
      right-control, confirmed it was working, then closed the
      connection)  The profile file was updated with today's date and
      time.
    
    Here are the contents:
    [remmina]
      sound=local,22050,2
      sharefolder=/home/kemp
      name=CITMS02
      cert_ignore=0
      console=0
      ssh_enabled=0
      exec=
      clientname=
      loadbalanceinfo=
      server=10.24.170.101
      colordepth=15
      ssh_auth=0
      postcommand=
      group=DSG - RDP
      ssh_charset=
      quality=1
      username=*****
      sharesmartcard=0
      ssh_loopback=0
      ssh_username=
      password=*****
      gateway_server=
      disablepasswordstoring=0
      microphone=0
      window_maximize=0
      execpath=
      gateway_usage=1
      viewmode=1
      resolution=1850x1050
      domain=*****
      window_height=1085
      security=
      precommand=
      window_width=1856
      ssh_server=
      shareprinter=0
      protocol=RDP
      disableclipboard=0
      ssh_privatekey=
      disableautoreconnect=0
    
    When I checked a profile for an RDP connection to a Windows 7
      box, I found the entry.  ( And it does seem to "stick" )
    I manually added the "keyboard_grab=1" entry to the profile for
      the Server 2012 profile and it seems to be working.  Not sure why
      it wasn't getting stored in there?  
    
    Anyway, works for me for now.  Thanks so much for your help.
    Shane
    
    
    On 11/07/2016 11:00 AM, Giovanni
      Panozzo wrote:
    
    
      @slcpunk: remmina profiles files are
        stored in $HOME/.local/share/remmina/ (or $HOME/.remmina for
        backward compatibility). They are files with the ".remmina"
        extension, and they should be updated when a connection is
        regularly closed (not when killing remmina).
        Try to find the correct .remmina file that corresponds to your
        connection, and open it with a text editor. You should find a
        line called "keyboard_grab=0" or 1. Check if it changes after
        connecting, enabling keyboard capture, and then disconnecting or
        closing the connection window.
      —
        You are receiving this because you were mentioned.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/FreeRDP/Remmina","title":"FreeRDP/Remmina","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/FreeRDP/Remmina"}},"updates":{"snippets":[{"icon":"PERSON","message":"@giox069 in #1041: @slcpunk: remmina profiles files are stored in $HOME/.local/share/remmina/ (or $HOME/.remmina for backward compatibility). They are files with the \".remmina\" extension, and they should be updated when a connection is regularly closed (not when killing remmina).\r\nTry to find the correct .remmina file that corresponds to your connection, and open it with a text editor. You should find a line called \"keyboard_grab=0\" or 1. Check if it changes after connecting, enabling keyboard capture, and then disconnecting or closing the connection window.\r\n"}],"action":{"name":"View Issue","url":"https://github.com/FreeRDP/Remmina/issues/1041#issuecomment-258911789"}}}

**Comment by giox069** [comment]
This problem is very similar to issue #1042, but we don't really know the cause. File permissions ?

**Comment by slcpunk** [comment]
Maybe...not convinced.  Profiles are saved in my home folder, and
      all have my user as owner.  I can't see a permission issue there. 
      Other settings are saved just fine, only the keyboard_grab is
      inconsistent.  
    
    
    
    
    
    
    On 11/07/2016 11:15 AM, Giovanni
      Panozzo wrote:
    
    
      This problem is very similar to issue #1042,
        but we don't really know the cause. File permissions ?
      —
        You are receiving this because you were mentioned.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/FreeRDP/Remmina","title":"FreeRDP/Remmina","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/FreeRDP/Remmina"}},"updates":{"snippets":[{"icon":"PERSON","message":"@giox069 in #1041: This problem is very similar to issue #1042, but we don't really know the cause. File permissions ?\r\n"}],"action":{"name":"View Issue","url":"https://github.com/FreeRDP/Remmina/issues/1041#issuecomment-258916139"}}}

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @giox069 @slcpunk
